package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "exif-js": "^2.3.0",
     "fit-file-parser": "^1.6.18",
+    "igc-parser": "^0.5.0",
```

Comment:
Could you also commit the changes to `package-lock.json`?

Reply:
I'm getting huge diffs for `package-lock.json`.

With `node=v15.5.0` and `npm=7.3.0` I'm getting a new file format and 10k lines of diffs: https://github.com/unbe/derive/commit/295815369b3e6e6f4e28289aed950c5ef4786fec
Note that `lockfileVersion` is updated from 1 to 2 there.

On another machine I have, with `node=v10.21.0` and `npm=5.8.0` I still get a large diff:
https://github.com/unbe/derive/commit/b3046ed800bf3e46f32159892fad762d48cf8100
Note that versions are getting more exactly specified there.

Do you want any of those in?

Alternatively, I can get node/npm version you're using and try to get a saner diff with those. If that's the way, let me know the versions.

Reply:
Hm, if the diff is that large it's possible that at some point I forgot to update the `package-lock.json` file myself. Let's ignore this for now, I'll clean this up separately.

Thank you!